bokehjs/src/coffee/renderer/glyph/image_url.coffee

```diff
@@ -23,15 +27,20 @@ class ImageURLView extends Glyph.View
 
       if need_load[i]
         img = new Image()
-        img.onload = do (img, i) =>
+        img.onerror = do (i, img, url) =>
+          return () =>
+            if @retry_attempts[i] > 0
+              setTimeout((-> img.src = url[i]), @mget('retry_timeout'))
+            @retry_attempts[i] -= 1
```

Comment:
I added the log statement: 

``ImageURL has exhausted retry_attempts and failed to load image``